mycore-base/src/main/java/org/mycore/datamodel/metadata/MCRMetaSpatial.java

```diff
@@ -0,0 +1,122 @@
+package org.mycore.datamodel.metadata;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Locale;
+import java.util.stream.Collectors;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import org.jdom2.Element;
+import org.mycore.common.MCRException;
+
+/**
+ * Stores spatial information for geographic references. The latitude longitude values are stored in an array list,
+ * where two BigDecimal build a point.
+ *
+ * @author Matthias Eichner
+ */
+public class MCRMetaSpatial extends MCRMetaDefault {
+
+    private List<BigDecimal> data;
```

Comment:
Two coordinates shouldn't be stored in a list. This should be two separate fields: lat and lng